src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarCommercial2.jest.tsx

```diff
@@ -66,4 +68,22 @@ describe("ArtworkSidebarCommercial RTL", () => {
 
     expect(screen.getAllByRole("radio").length).toBe(4)
   })
+
+  it("displays 'Taxes may apply at checkout.'", () => {
```

Comment:
For context: removed the old tests and replaced them with RTL tests which are living on `ArtworkSidebarCommercial2.jest.tsx`